### DISCUSSION TITLE
Open file into suspended instance of Helix

### DISCUSSION THREAD

**Discussion by cem-al** [helix-editor/helix#5916] [Q&A]
Can I suspend helix 'ctrl-z' and then open a file from the shell with that suspended instance?

Instead of loading a new instance?

**Comment by michaeljones** [comment]
I have a patched version of 22.12 where I've added a socket to receive open commands. It isn't really an out of the box solution but I think if 'ctrl-z' just backgrounds the process then it should work. Though it won't foreground the process, you'll have to foreground it after: https://github.com/helix-editor/helix/discussions/5107#discussioncomment-4549555
